tests/NHSD.BuyingCatalogue.Testing.Data/Entities/SolutionEntity.cs

```diff
@@ -6,7 +6,9 @@ namespace NHSD.BuyingCatalogue.Testing.Data.Entities
 {
     public sealed class SolutionEntity : EntityBase
     {
-        public string Id { get; set; }
+        public string SolutionId { get; set; }
+
+        public int PublishedStatusId { get; set; }
```

Comment:
We shouldn't need this here. The published status is recorded at the catalogue item level.

Reply:
Removed `PublishedStatusId`